### ISSUE TITLE
Language links are 404

### ISSUE THREAD

**Issue by aviaryan** [resume/resume.github.com#201]
Links like [AutoHotkey](https://github.com/languages/AutoHotkey) shown under the languages category are 404.

```
Current: https://github.com/languages/AutoHotkey
Correct: https://github.com/trending?l=autohotkey
```

**Comment by Walkman100** [comment]
:+1:

**Comment by bfontaine** [comment]
This is fixed by #180.

**Comment by u2** [comment]
Fix by #219
